deploy/project.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# composer update
+cd /vagrant
+composer global require "fxp/composer-asset-plugin:1.0.0"
```

Comment:
That was just for composer/bower integration. Not really needed since we don't have bower assets here.